zipline/utils/preprocess.py

```diff
@@ -172,21 +171,29 @@ def make_processor_assignment(arg, processor_name):
     signature = []
     call_args = []
     assignments = []
+    star_map = {
+        varargs: '*',
+        varkw: '**',
+    }
+
+    def argname(arg):
```

Comment:
I might call this `name_as_arg` to help clarify that this is the name as it will appear in the signature, as distinguished from the name as it will be used in the body of the function.